crates/win-api-wrappers/src/win.rs

```diff
@@ -0,0 +1,3683 @@
+use std::{
```

Comment:
Lets split this file to multiple small modules, single with >3K lines of code is very hard to follow 😢